src/utils/map-styling/transit.ts

```diff
@@ -15,23 +16,17 @@ import {
   StyleTypes,
 } from '../../utils/applyStyle';
 
-const AirportLayerNames = [
-  'airport-label',
-  'aeroway-polygon',
-  'transit-airport',
-  'aeroway-line',
+const PolygonLayers = [
+  'mapbox-airport-aeroway-polygon',
+  'mapbox-airport-polygon',
 ];
-const BusLayerNames = ['transit-bus-label'];
-const RailLayerNames = ['transit-rail'];
-const SubwayLayerNames = ['transit-subway', 'transit-subway-label'];
-
-const AllLayerTypeKeys = [
-  ...AirportLayerNames,
-  ...BusLayerNames,
-  ...RailLayerNames,
-  ...SubwayLayerNames,
-  'transit-label',
+const LineLayers = [
+  'mapbox-airport-aeroway-line',
+  'transit-subway-line',
+  'transit-rail-line',
+  'mapbox-rail-road-line',
 ];
+const LabelLayers = ['mapbox-airport-label', 'transit-bus-label'];
```

Comment:
궁금한게 있습니다!
 layer 이름을 항상 이렇게 전부 일일이 매핑/정의를 해놓아야하나요?

수작업으로 하다보니 실수할우려도 있고, 추가/변경 시 작업하기 쉽지 않을 거같습니다.
구분 규칙이 없어서 방법이 없다면 어쩔수없지만, 개선이 되면 좋긴하겠네요 :)
(예를들어, layer type 별로 필터링이 가능한지, 아예 style/layer 구성 시 meta 정보를 함께 구성해 이 정보로 필터링을 한다던지요~)

Reply:
지나가던 중 답변을 잠시 드리자면, 현재 rendering-data/layerColor(1-4)로 색상정리를 하면서 조금이나마 정리를 하고있습니다. 추후에 이것을 좀 더 정리한 후 발전시키면 가능할 것으로 보입니다! 한번 논의해 보겠습니다!

Reply:
멘토님 지적해주신대로 실수를 몇번 했어서 수작업이 아닌, 개선을 해야 할 것 같습니다.

간단하게는, `map/layers/카테고리`에서 배열로 export를 하고 있기 때문에
아래와 같이 id string을 가지고 filter를 적용하면 될 것 같습니다.

```suggestion
import transitLayers from ../../store/map/layers/transit

const LabelLayers = transitLayers.filter(layer => layer.contains('label'));
```

모든 세부 메뉴-스타일 메뉴 마다 전체 layer 정보를 다 갖고 있는 것도 성능에 영향을 미칠 수 있을지 고민해봐야 할 것 같습니다